src/main/resources/swagger-static/index.html

```diff
@@ -76,13 +76,23 @@
       function addApiKeyAuthorization(){
         var key = encodeURIComponent($('#input_apiKey')[0].value);
         if(key && key.trim() != "") {
-            var apiKeyAuth = new SwaggerClient.ApiKeyAuthorization("api_key", key, "query");
+            var apiKeyAuth = new SwaggerClient.ApiKeyAuthorization("x-auth-id", key, "header");
             window.swaggerUi.api.clientAuthorizations.add("api_key", apiKeyAuth);
             log("added key " + key);
         }
       }
 
+      function addApiSecretAuthorization(){
+        var secret = encodeURIComponent($('#input_apiSecret')[0].value);
+        if(secret && secret.trim() != "") {
+            var apiSecretAuth = new SwaggerClient.ApiKeyAuthorization("x-auth-key", secret, "header");
```

Comment:
I think `var` should be declared at the beginning of the scope and `if` do not define scope